### PR TITLE
Fixing linear regression, cloning models and adapting tests

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

**Comment by mmerce** [comment]
Finally, tests passed. Ready for review.

File: docs/index.rst [hunk 1]
```diff
@@ -109,6 +114,8 @@ those variables automatically when you log in:
     export BIGML_USERNAME=myusername
     export BIGML_API_KEY=ae579e7e53fb9abd646a6ff8aa99d4afe83ac291
 
+refer to the next chapters to know how to do that in other operative systems.
```
Comment (jaor) [inline docs/index.rst]: typo: operating

File: docs/index.rst [hunk 2]
```diff
@@ -3596,15 +3628,29 @@ in the previous section:
...
-users can define some variables in the code to be exported as outputs. Please
-refer to the
-`Developers documentation for Executions <https://bigml.com/api/executions#we_execution_arguments>`_
-for details on how to define execution outputs.
+users can define some variables in the code to be exported as outputs.
+
+An ``execution`` receives inputs, the ones defined in the ``script`` chosen
+to be executed, and generates a result. It can also generate outputs. To
+execute a ``script`` that expects some inputs, you will need to specify the
+concrete values of these inputs, unless a default value has been assigned
```
Comment (jaor) [inline docs/index.rst]: silly nit: those inputs

File: docs/index.rst [hunk 3]
```diff
@@ -3596,15 +3628,29 @@ in the previous section:
...
+for them in the script inputs definition. Following the second example in
```
Comment (jaor) [inline docs/index.rst]: script's

**Pull request merged**